### PR TITLE
[ADD][12.0] product_uom_measure_type module

### PR DESCRIPTION
**Pull request by legalsylvain**
Following the remark from @hparfr, (https://github.com/OCA/product-attribute/pull/852/files#r623027969) clean the module ``product_uom_po_domain`` and create a dedicated module to have a the field ``measure_type`` on product template and variant models.

### CODE REVIEW

File: product_uom_measure_type/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    uom_measure_type = fields.Selection(
+        string="UoM Measure Type",
+        related="uom_id.category_id.measure_type", store=True)
```
Comment (hparfr) [inline product_uom_measure_type/models/product_template.py]: Does this double dotted introduce a perf penalty ?

**Review by hparfr — APPROVED** [review]
I just have a non-blocking question.
Otherwise LGTM

File: product_uom_measure_type/models/product_template.py (hunk 1 above)
Comment (legalsylvain) [reply product_uom_measure_type/models/product_template.py]: I really don't know ! 

Well, checking again, I saw that measure_type was present in uom.uom model. (related, store=True). So, changed for a single dotted relation.

File: product_uom_po_domain/readme/DEVELOP.rst [hunk 2]
```diff
@@ -1,5 +1,3 @@
 Technically, this module add two new related fields on ``product.template`` model,
```
Comment (remi-filament) [inline product_uom_po_domain/readme/DEVELOP.rst]: There is only 1 left no ? :)

**Review by remi-filament — APPROVED** [review]
LGTM, 1 minor comment

**Comment by legalsylvain** [comment]
Hi @OCA/product-maintainers : could you merge this PR ?

**Pull request merged**